src/app/components/PageLayout/index.tsx

```diff
@@ -37,12 +37,24 @@ export const PageLayout: FC<PropsWithChildren<PageLayoutProps>> = ({ children, m
       >
         <Header />
         <Box
-          sx={{
-            border: isMobile ? 'none' : `solid 15px ${theme.palette.layout.border}`,
-            borderTop: 0,
-            px: isMobile ? 0 : '4%',
-            pt: isMobile ? 4 : 6,
-          }}
+          sx={
+            isMobile
+              ? {
+                  border:
+                    theme.palette.background.default !== theme.palette.layout.border
```

Comment:
No, because when we don't want the see border, then we don't want to have it at all; when it's there, it's really thick, and so it's a big waste of space to have an invisible border. Plus there are other changes, including the different border radius...